### DISCUSSION TITLE
Parameter format

### DISCUSSION THREAD

**Discussion by Mephisto090** [mcuiobroker/docuJarvis#7] [Q&A] [answered]
Hej mcu,

jetzt haben die Spalten ja keine Namen wie mache ich jetzt null zu einem minus? :)

```
{
  "field": "Plattform",
  "format": "val => val === null ? '-' : val"
}
```

```
<table width="100%">
<tr>
  <th align="left" colspan="4">A - B</th>
</tr>
<tr>
  <th align="center">Abfahrt</th>
  <th align="center">Verspätung</th>
  <th align="center">Plattform</th>
  <th align="center">Ankunft</th>
  <th align="center">Verspätung</th>
  <th align="center">Plattform</th>
</tr>
<tr>
<td>12:13</td>
<td>0</td>
  <td><font color="#008000">null</font></td>
  <td><font color="#008000">12:59</font></td>
  <td><font color="#008000">0</font></td>
  <td><font color="#008000">null</font></td>
</tr>
<tr>
  <td>13:13</td>
  <td>0</td>
  <td><font color="#008000">null</font></td>
  <td>14:09</td>
  <td>0</td>
  <td><font color="#008000">null</font></td>
</tr>
<tr>
  <td>14:13</td>
  <td>0</td>
  <td><font color="#008000">null</font></td>
  <td>15:09</td><td>0</td>
  <td><font color="#008000">null</font></td>
</tr>
</table>
```

![image](https://user-images.githubusercontent.com/85995176/194033561-9dc55172-d085-4c52-b678-c5e05dc41dc3.png)

Mir fällt gerade auf Farbe wird gar nicht angezeigt

**Comment by mcuiobroker** [comment, marked as answer]
In diesem Fall ist es ein String -> 'null'
`[{   "field": "Plattform",   "format": "val => val === 'null' ? '-' : val" }]`

**Reply by Mephisto090** [reply]
Ist mir auch gerade Aufgefallen. Geht auch nicht :(

**Comment by mcuiobroker** [comment]
Welche Version ? v3.0.11?
mit v3.1.0-alpha.54 sieht es so aus:
![image](https://user-images.githubusercontent.com/84818934/194037484-9e234dde-2ef7-4d99-88a3-0f142dc23a59.png)

**Reply by Mephisto090** [reply]
Ja die eckigen Klammern haben bei mir gefehlt :D

**Comment by Mephisto090** [comment]
> Mir fällt gerade auf Farbe wird gar nicht angezeigt

Weißt du woran das liegen kann?

**Reply by mcuiobroker** [reply]
PRO-Version?

**Reply by Mephisto090** [reply]
Na sicher doch :D

**Reply by mcuiobroker** [reply]
Dann scheint es noch ein Fehler im Module HtmlTable zu geben?

**Reply by Mephisto090** [reply]
Ich schreibe mal Zefau

**Reply by mcuiobroker** [reply]
Direkt als Bug ein Issue aufmachen.

**Reply by Mephisto090** [reply]
Jap so wie immer :)

**Reply by Mephisto090** [reply]
Ist drin, kannst ja mal schauen
